tests/testthat/test-make-style.r

```diff
@@ -30,13 +30,13 @@ test_that("hexa color regex works", {
                 "foo#123456bar")
 
   for (color in positive) {
-    expect_true(grepl(hash_color_regex, color))
-    expect_true(grepl(hash_color_regex, toupper(color)))
+    expect_true(grepl(crayon:::hash_color_regex, color))
+    expect_true(grepl(crayon:::hash_color_regex, toupper(color)))
```

Comment:
I think this is fine without `:::` as long as you run the tests via `testthat`? No?